studentska/osobe.php

```diff
@@ -1093,7 +1093,32 @@ function studentska_osobe() {
 		if (!$dry_run) {
 			if (!$ponovac) $ponovac=0;
 			if (!$plan_studija) $plan_studija=0;
-			db_query("insert into student_studij set student=$student, studij=$studij, semestar=$semestar, akademska_godina=$godina, nacin_studiranja=$nacin_studiranja, ponovac=$ponovac, odluka=NULL, plan_studija=$plan_studija");
+			$apsolvent = 0;
+
+			// Postavljanje uslova da li je student apsolvent ili nije !
+            // Prvi od uslova je da li se student upisuje u 5-6 ili 9-10 semestar
+            if($semestar == 5 or $semestar == 6 or $semestar == 9 or $semestar == 10){
```

Comment:
Treba u uslov dodati i ciklus. Što se tiče 9 i 10 semestra, oni se u zamgeru ne vode kao semestar 9 i 10 nego kao semestar 3 i 4 ali je $ciklus == 2